include/libssh/session.h

```diff
@@ -106,6 +106,15 @@ struct ssh_common_struct {
     int log_verbosity; /* verbosity of the log functions */
 };
 
+enum ssh_socket_states_e {
```

Comment:
This is used on line 126 below, but because it was not declared here before, it was considered a forward declaration. clang balked at this.